### DISCUSSION TITLE
New module proposal for @turf/bezier-curve

### DISCUSSION THREAD

**Discussion by DenisCarriere** [Turfjs/turf#2780] [Feature proposals]
Also it might be good to have two separate modules for `Bezier` operations:

- `@turf/bezier-curve`
- `@turf/bezier-spline`

## Bezier Curve

Propose for new module `@turf/bezier-curve`.

**Demo**: https://www.desmos.com/calculator/cahqdxeshd

**Javascript Implementation:** https://github.com/Pomax/bezierjs

![image](https://cloud.githubusercontent.com/assets/550895/24911479/0fd179d2-1e99-11e7-81a8-ba9c29c136f8.png)

![image](https://cloud.githubusercontent.com/assets/550895/24912021/dfed1526-1e9a-11e7-86a0-84f753773e55.png)

## Bezier Spline

Current implementation of `@turf/bezier` could be renamed to `@turf/bezier-spline` to be more explicit (if so major release is needed).

![image](https://cloud.githubusercontent.com/assets/550895/24911457/f7083d32-1e98-11e7-9df8-cabdad241267.png)

**Comment by DenisCarriere** [comment]
I'll never finish module if I keep bumping it up to the next release :) lol Ah well.. next release!

**Comment by eric-g-97477** [comment]
I just wanted to bump this issue again. Better bezier support would be useful. It could, perhaps, be based off of or integrated with https://github.com/Pomax/bezierjs which provides good bezier support.
